dracut/28immucore/kairos-setup-initramfs.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=kairos system initramfs setup before switch root
+DefaultDependencies=no
+After=initrd-fs.target
+Requires=initrd-fs.target
+Before=initrd.target
+
+[Service]
+RootDirectory=/sysroot
```

Comment:
I'm ok with this atm, but just a note for the future, we could call it directly from immucore and done (otherwise other init systems need special hooks for it too..)

Reply:
yes, this is clearly marked on several places that needs to go into immucore

Reply:
you know I'm a broken disk :notes: